Makefile.in

```diff
@@ -405,6 +408,9 @@ perf$(EXEEXT):	$(PERF_OBJ_LINK)
 delayer$(EXEEXT):	$(DELAYER_OBJ_LINK)
 	$(LINK) -o $@ $(DELAYER_OBJ_LINK) $(SSLLIB) $(LIBS)
 
+readzone$(EXEEXT):	$(READZONE_OBJ_LINK)
```

Comment:
Add readzone to TEST_BIN.

Reply:
resolved in https://github.com/NLnetLabs/unbound/pull/491/commits/3d50c25f5b4d709f3fc1c7229b232289af48ed24